bindings/python/tests/test_output.py

```diff
@@ -237,5 +237,5 @@ def test_output():
             }
         ]
     }
-    nft_output = from_dict(NftOutput, nft_output_dict)
-    assert nft_output.as_dict() == nft_output_dict
+    nft_output = NftOutput.from_c, nft_output_dict)
```

Comment:
Weird

Reply:
Because tests are disabled for 2.0

Reply:
Hmm ok, I thought they would be compiled like the examples at least, so the syntax is checked